util/postman/openkit.postman_collection.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
🔥🔥🔥! Out of curiosity, did you generate the same way we exported postman collections in the past?

Reply:
Correct. The exact same way.